docs/partner_editable/deploy_steps.adoc

```diff
@@ -44,7 +44,9 @@ Each deployment takes about {deployment_time} to complete.
 
 [start=3]
 . On the *Create stack* page, keep the default setting for the template URL, and then choose *Next*.
-. On the *Specify stack details* page, change the stack name if needed. Review the parameters for the template. Provide values for the parameters that require input. For all other parameters, review the default settings and customize them as necessary.
+. On the *Specify stack details* page, change the stack name if needed. Review the parameters for the template. Provide values for the parameters that require input. For all other parameters, review the default settings and customize them as necessary. See the "Parameter reference" section.
+
+//TO DO Shivansh, How do we add a link to "Parameter reference" in the previous sentence? (After we add the link, we'll remove the quotation marks.)
```

Comment:
I think we can directly link it using standard anchor tag like <<>>.

Reply:
@sshvans The standard <<>> anchor tag you note here italicizes the link. Since that's not AWS style, I used another approach: `See the link:#_parameter_reference[Parameter reference] section.`